.azure-pipelines/gpu-benchmark.yml

```diff
@@ -36,7 +36,7 @@ jobs:
 
     steps:
       - bash: |
-          python -m pytest benchmarks -v --durations=0
+          python -m pytest tests/benchmarks -v --durations=0
```

Comment:
Is it not possible to keep the benchmarks folder separate?

Reply:
Nope

Reply:
in such case we shall rather make it stand-alone and eventually outsource in compatibility repo

Reply:
Sure, it's probably going to be changed after the community repo is fully formalized and our benchmarks review.

It can be done as a follow-up